src/app/templates/_README.md

```diff
@@ -0,0 +1,40 @@
+# <%= nameOf.package %>
+
+A plugin for video.js. 
```

Comment:
Lost the insertion of the plugin description here, compared to the previous version.